dvc/analytics.py

```diff
@@ -1,256 +1,131 @@
-"""Collect and send usage analytics"""
-from __future__ import unicode_literals
-
-import errno
 import json
 import logging
-import os
+import platform
+import requests
+import sys
+import tempfile
+import uuid
 
-from dvc import __version__
-from dvc.utils import env2bool
-from dvc.utils.compat import str
+import distro
 
+from dvc import __version__
+from dvc.config import Config, to_bool
+from dvc.daemon import daemon
+from dvc.exceptions import NotDvcRepoError
+from dvc.lock import Lock, LockError
+from dvc.repo import Repo
+from dvc.scm import SCM
+from dvc.utils import env2bool, is_binary, makedirs
 
 logger = logging.getLogger(__name__)
 
 
-class Analytics(object):
-    """Class for collecting and sending usage analytics.
-
-    Args:
-        info (dict): optional existing analytics report.
+def collect_and_send_report(arguments=None, exit_code=None):
```

Comment:
```suggestion
def collect_and_send_report(arguments=None, return_code=None):
```
We are passing ret to it and then passing it to cmd_return_code, it is weird to have `exit_code` in between.

Reply:
that's true, @efiop